operations/UAS_user_and_admin_topics/Customize_End-User_UAI_Images.md

```diff
@@ -47,7 +47,7 @@ A custom end-user UAI image can be any container image set up with the end-user
     name: wlm-sessiontemplate-0.1.0
```

Comment:
The version of COS being shipped with Shasta v1.5 has greatly reduced the number of kernel_parameters that need to be listed in the session template.  We updated the COS examples to only show this:
        "kernel_parameters": "ip=dhcp quiet spire_join_token=${SPIRE_JOIN_TOKEN}",

Just a heads up in case you want to update the examples in the CSM docs as well.